lib/wordpress.js

```diff
@@ -0,0 +1,48 @@
+var _ = require('underscore');
+var params = require('querystring').parse(window.location.search.split('?').pop());
+
+module.exports = {
+  getPostPreviewView: function (view) {
+    // Return false if we don't have everything we need for a preview
+    var isPreviewActive = params._wp_json_nonce && params.is_preview;
+    if (!isPreviewActive) {
+      console.info('Preview is not active.');
+      return false;
+    }
+
+    // Create preview object
+    if (_.isFunction(view)) {
+      view = {template: view};
+    }
+    if (_.isFunction(view.preprocessor)) {
```

Comment:
Wouldn't hurt to throw just in case if someone did though, telling them what's required rather than silently failing.